### DISCUSSION TITLE
Alguém tem dicas de onde encontrar APIs legais ? é pro meu TCC =D

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1855] [Perguntas e Respostas]
Queria saber de vocês. APIs que acham curiosas, interessantes ou até não tão famosas mas que te chamaram a atenção.

**Comment by mstuttgart** [comment]
Fala @romulosardinha , tudo certo?

Lembrei de duas que acho bem legais:

* https://jsonplaceholder.typicode.com/ - API fake pra testes
* https://api.chucknorris.io/ - API com piadas do Chuck Norris

Ambas são muito boas pra estudos!

**Comment by mthalesb** [comment]
Nesse repositório tem umas interessantes: https://github.com/public-apis/public-apis

**Comment by guilhermesantoss** [comment]
Eae @romulosardinha, recentemente fiz um teste pra uma empresa consumindo a seguinte API:
- https://restcountries.eu/ 

Bem legal e da pra fazer bastante coisa, espero ter ajudado!

**Comment by AmandaFonseca** [comment]
ac
